pkgs/development/python-modules/jxlpy/default.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  cython,
+  libjxl,
+}:
+buildPythonPackage rec {
+  pname = "jxlpy";
+  version = "0.9.5";
+
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-Kqdm8b3hgO0Q3zE68rSIM4Jd7upjG+SQywSseGwCFUI=";
+  };
+
+  build-system = [
+    setuptools
+    cython
+  ];
+
+  buildInputs = [ libjxl ];
+
+  pythonImportsCheck = [ "jxlpy" ];
+
```

```suggestion
  # no tests
  doCheck = false;
```
We highly recommend enabling tests, so if there are no tests, could you explicitly disable them?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#testing-python-packages-testing-python-packages